
Turn 5
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 127, in <module>
    move(ship, ship_paths[ship.id])
